### PR TITLE
Html and the web personal website

### PR DESCRIPTION
**Pull request by rubenskuhn**

### CODE REVIEW

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:6]: You should add a title to your page

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  </head>
+
+    <body style="background-color:rgb(255, 165, 127);"></body>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:9]: nice touch; in the future this should go into a css stylesheet, not directly in the HTML-file

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ 
+  
+
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:12]: you could add a <main>-element to show where the main content of the page is

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <h1>Rubens Kuhn</h1>
+    <p>The Noob of Coders: He may look retarded,</p>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:14]: you don't have to add a closing tag for the paragraph (< /p>) after every line; if you were going for a line break, you can use < br>
*both without the space after "<" of course

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    <p>but can code the sh#t out of you while doing so!</p>
+    <p></p>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:16]: empty paragraph, you can delete this to keep your code clean; again: line break with < br>

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    <img src="../images/061.jpg" width="500" "Looking Stoopid ;)" />
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:17]: before the "looking stoopid ;)" there is "alt=" missing

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    <p>Looking Stoopid ;)</p>
+    <h2>Contacts</h2>
+    <p>You know what to do!</p>
+    <li>Tel: 101018303493921</li>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:21]: the li-element alone doesn't work, you need a ul- or ol-element around it

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    <li>Somewhere in Berlin</li>
+    <p>              </p>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:23]: I feel like this is the line break topic again :) see above

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+    <h3>Or just press some butons!</h3>
+    <p></p>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:25]: I feel like this is the line break topic again :) see above

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+    <a href="https://www.linkedin.com/in/rubenskuhn/"></a><img src="../images/WhatsApp.png" width="30" "WhatsApp"</a>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:26]: there's a a-closing tag in the middle that you dont need :)

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+    <p></p>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:27]: I feel like this is the line break topic again :) see above

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+    <a href="https://www.linkedin.com/in/rubenskuhn/"></a><img src="../images/Linkedin.png" width="30" "WhatsApp"</a>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:28]: closing a-tag too much in the middle again

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+    <p></p>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:29]: I feel like this is the line break topic again :) see above

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+    <p></p>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:30]: I feel like this is the line break topic again :) see above

**Review by itsTHL — CHANGES_REQUESTED** [review]
left you some comments :)

File: html-and-web/html-and-the-web_personal-website/index.html (hunk 4 above)
Comment (rubenskuhn) [reply html-and-web/html-and-the-web_personal-website/index.html:14]: Thanks!

File: html-and-web/html-and-the-web_personal-website/index.html (hunk 1 above)
Comment (rubenskuhn) [reply html-and-web/html-and-the-web_personal-website/index.html:6]: Done

File: html-and-web/html-and-the-web_personal-website/index.html (hunk 2 above)
Comment (rubenskuhn) [reply html-and-web/html-and-the-web_personal-website/index.html:9]: noted :)

**Pull request merged**